schema/me/conversation/message.js

```diff
@@ -84,12 +89,26 @@ export const MessageType = new GraphQLObjectType({
       type: new GraphQLList(AttachmentType),
     },
 
+    invoice: {
+      type: InvoiceType,
+      resolve: ({ metadata, conversation_id }, options, request, { rootValue: { conversationInvoiceLoader } }) => {
+        if (!isInvoiceMessage(metadata)) {
+          return null
+        }
+        return conversationInvoiceLoader({
+          conversation_id,
+          lewitt_invoice_id: metadata.lewitt_invoice_id,
+        }).then(invoice => {
+          return invoice
+        })
+      },
+    },
+
     is_invoice: {
       description: "True if message is an invoice message",
+      deprecationReason: "Prefer to use invoice.",
```

Comment:
I’m not sure this needs to be deprecated. It could still be legitimate to just check if this message is for an invoice, no? Having to use the `invoice` field would mean you’d have to pick an arbitrary field inside the `invoice` object to make a valid query.